src/ray/gcs/gcs_server/gcs_table_storage.h

```diff
@@ -0,0 +1,373 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RAY_GCS_GCS_TABLE_STORAGE_H_
+#define RAY_GCS_GCS_TABLE_STORAGE_H_
+
+#include <utility>
+
+#include "ray/gcs/store_client/redis_store_client.h"
+#include "ray/protobuf/gcs.pb.h"
+
+namespace ray {
+namespace gcs {
+
+using rpc::ActorCheckpointData;
+using rpc::ActorCheckpointIdData;
+using rpc::ActorTableData;
+using rpc::ErrorTableData;
+using rpc::GcsNodeInfo;
+using rpc::HeartbeatBatchTableData;
+using rpc::HeartbeatTableData;
+using rpc::JobTableData;
+using rpc::ObjectTableData;
+using rpc::ObjectTableDataList;
+using rpc::ProfileTableData;
+using rpc::ResourceMap;
+using rpc::ResourceTableData;
+using rpc::TaskLeaseData;
+using rpc::TaskReconstructionData;
+using rpc::TaskTableData;
+using rpc::WorkerFailureData;
+
+#define JOB_TABLE_NAME "JOB";
+#define ACTOR_TABLE_NAME "ACTOR";
+#define ACTOR_CHECKPOINT_TABLE_NAME "ACTOR_CHECKPOINT";
+#define ACTOR_CHECKPOINT_ID_TABLE_NAME "ACTOR_CHECKPOINT_ID";
+#define TASK_TABLE_NAME "TASK";
+#define TASK_LEASE_TABLE_NAME "TASK_LEASE";
+#define TASK_RECONSTRUCTION_TABLE_NAME "TASK_RECONSTRUCTION";
+#define OBJECT_TABLE_NAME "OBJECT";
+#define NODE_TABLE_NAME "NODE";
+#define NODE_RESOURCE_TABLE_NAME "NODE_RESOURCE";
+#define HEARTBEAT_TABLE_NAME "HEARTBEAT";
+#define HEARTBEAT_BATCH_TABLE_NAME "HEARTBEAT_BATCH";
+#define ERROR_INFO_TABLE_NAME "ERROR_INFO";
+#define PROFILE_TABLE_NAME "PROFILE";
+#define WORKER_FAILURE_TABLE_NAME "WORKER_FAILURE";
```

Comment:
Just use the `TablePrefix` enum in protobuf.

Reply:
fixed